tests/test_benchmark_scenarios.py

```diff
@@ -327,3 +331,23 @@ def test_benchmark_AFG_grid_heatpump_heat(self, margs):
     def teardown_method(self):
         if os.path.exists(TEST_OUTPUT_PATH):
             shutil.rmtree(TEST_OUTPUT_PATH, ignore_errors=True)
+
+
+# this ensure that the test is only ran if explicitly executed, ie not when the `pytest` command
```

Comment:
Is that what we always write? Because the tests also runs when you set `EXECUTE_TESTS_ON =master`, so I find the last half misleading
```suggestion
# this ensure that the test is only ran if explicitly executed

Reply:
Yeah I copy pasted it, we should probably all revisit those comments ...